stdlib/3/_importlib_modulespec.pyi

```diff
@@ -9,6 +9,8 @@ from abc import ABCMeta
 import sys
 from typing import Any, Optional
 
+MYPY_MODULE = 'importlib.machinery'
```

Comment:
I don't have an opinion on this, I would be happy to hard code it in mypy. Just wasn't sure if it's ok given that it's hard to find later when more circular import definitions happen.